test-util/config.ts

```diff
@@ -31,16 +31,14 @@ export class Config
 
 export async function config(props?: NodeProps): Promise<Config> {
   if ("_browserShim" in Deno) return new Config(9944, () => {});
-  let port = 9944;
+  let port;
   if (props?.port) {
     if (!isPortAvailable(props.port)) {
       fail(`Port ${props.port} is unavailable`);
     }
     port = props.port;
   } else {
-    while (!isPortAvailable(port)) {
-      port++;
-    }
+    port = getRandomPort();
```

Comment:
@harrysolovay added a random port so it's less likely to have the race condition in parallel tests